nav/nav_master/nav_master.py

```
@@ -183,6 +182,7 @@ def compute_star_models(self) -> None:
         self._star_models = [stars_model]
         self._metadata['models']['star_model'] = stars_model.metadata
 
+        # import matplotlib.pyplot as plt  # noqa: F401
```

Ruff (0.14.7)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)